.rustfmt.toml

```diff
@@ -1,5 +1,6 @@
 # Basic
 hard_tabs = false
+tab_spaces = 4
```

Comment:
Based on the rustfmt docs, this is the default value. If so, do we need to specify it?

Reply:
In most cases I think omitting the defaults is the right way to go. For options that are part of our coding style, adding it makes it more visible and saves the need to check the documentation.